tests/distributions/test_multivariate.py

```diff
@@ -1709,6 +1709,14 @@ class TestDirichlet(BaseTestDistributionRandom):
         "check_rv_size",
     ]
 
+    @pytensor.config.change_flags(warn_float64="raise", floatX="float32")
+    def test_dirichlet_float32(self):
+        """https://github.com/pymc-devs/pymc/issues/6779"""
+        with pm.Model() as model:
+            c = pm.floatX([1, 1, 1])
+            pm.Dirichlet("a", c)
+        model.point_logps()
```

Comment:
Should be enough to request the `model.logp()`? No point in compiling and evaluating once if you're not checking the results